Flaky heartbeat unit test

<!-- This form is for bug reports and feature requests ONLY!
Also make sure that you visit our User Guide at https://kubevirt.io/user-guide/
-->

**Is this a BUG REPORT or FEATURE REQUEST?**:

> Uncomment only one, leave it on its own line:
>

/kind bug

**What happened**:
The "Heartbeat without deviceplugin and becoming ready after a few probes, node should be set to unschedulable immediately and switch earlier than one minute [It]" **unit** test is lately very flaky, as can be seen on [this CI kubevirt search](https://search.ci.kubevirt.io/?search=%2F%2Fpkg%2Fvirt-handler%2Fheartbeat%3Ago_default_test.*FAILED&maxAge=336h&context=1&type=build-log&name=&excludeName=&maxMatches=5&maxBytes=20971520&groupBy=job). 

It fails with the following trace:

```
------------------------------
• Failure [1.513 seconds]
Heartbeat
pkg/virt-handler/heartbeat/heartbeat_test.go:26
  without deviceplugin and
  vendor/github.com/onsi/ginkgo/extensions/table/table.go:92
    becoming ready after a few probes, node should be set to unschedulable immediately and switch earlier than one minute [It]
    vendor/github.com/onsi/ginkgo/extensions/table/table_entry.go:43

    Timed out after 1.000s.
    Expected
        <map[string]string | len:2>: {
            "cpumanager": "false",
            "kubevirt.io/schedulable": "false",
        }
    to have {key: value}
        <map[interface {}]interface {} | len:1>: {
            "kubevirt.io/schedulable": "true",
        }

    pkg/virt-handler/heartbeat/heartbeat_test.go:121
------------------------------
{"component":"","level":"info","msg":"Updating cluster config from KubeVirt to resource version 'sgjv7btdcj'","pos":"config-map.go:516","timestamp":"2021-06-01T16:21:40.827643Z"}

JUnit report was created: /tmp/cache/bazel/18316b1300bb8985bc913139d5cc6323/sandbox/linux-sandbox/4153/execroot/__main__/bazel-out/k8-fastbuild/testlogs/pkg/virt-handler/heartbeat/go_default_test/test.xml


Summarizing 1 Failure:

[Fail] Heartbeat without deviceplugin and [It] becoming ready after a few probes, node should be set to unschedulable immediately and switch earlier than one minute 
pkg/virt-handler/heartbeat/heartbeat_test.go:121

Ran 8 of 8 Specs in 2.543 seconds
FAIL! -- 7 Passed | 1 Failed | 0 Pending | 0 Skipped
--- FAIL: TestHeartbeat (2.54s)
FAIL
```

**What you expected to happen**:
The test must be deterministic and consistently pass.

**How to reproduce it (as minimally and precisely as possible)**:
Run unit tests. Odds are it'll blow up on your face.

**Anything else we need to know?**:
Not right now.

--- a/pkg/virt-handler/heartbeat/heartbeat_test.go
+++ b/pkg/virt-handler/heartbeat/heartbeat_test.go
@@ -92,7 +92,7 @@ var _ = Describe("Heartbeat", func() {
 
 	table.DescribeTable("without deviceplugin and", func(deviceController device_manager.DeviceControllerInterface, initiallySchedulable string, finallySchedulable string) {
 		heartbeat := NewHeartBeat(fakeClient.CoreV1(), deviceController, config(), "mynode")
-		heartbeat.devicePluginWaitTimeout = 1 * time.Second
+		heartbeat.devicePluginWaitTimeout = 2 * time.Second
 		heartbeat.devicePluginPollIntervall = 10 * time.Millisecond
 		stopChan := make(chan struct{})
 		done := heartbeat.Run(100*time.Second, stopChan)

